### PR TITLE
Update to 1.0.0 for blade-ui-kit/blade-icons

### PR DESCRIPTION
**Pull request by mallardduck**

### CODE REVIEW

**Comment by mallardduck** [comment]
hoping that after this PR is merged you can tag a new release too. 🙏

**Comment by Gummibeer** [comment]
This PR is needed for all projects depending on multiple Blade Icon packs or using packages like https://github.com/laravel-filament/filament .
Would be great if this can be merged and released. 🥳

File: composer.json
Comment (Gummibeer) [inline composer.json]: Blade icons dropped PHP 7.3 support with v1.0
> Drop PHP 7.3
> - https://github.com/blade-ui-kit/blade-icons/blob/1.x/CHANGELOG.md#100-2021-03-21

```suggestion
        "php": "^7.4|^8.0",
```

File: composer.json
Comment (Gummibeer) [inline composer.json]: Blade icons dropped Laravel 7 support with v1.0
> Drop Laravel 7 support
> - https://github.com/blade-ui-kit/blade-icons/blob/1.x/CHANGELOG.md#100-2021-03-21


```suggestion
        "illuminate/support": "^8.0"
```

`orchestra/testbench` v5.0 should also be removed from dev dependencies because of that.

**Review by Gummibeer — CHANGES_REQUESTED** [review]

**Comment by mallardduck** [comment]
Thanks @Gummibeer - those changes are in the PR now.
also normalized composer for extra fun.

**Review by Gummibeer — APPROVED** [review]

**Comment by scrlkx** [comment]
Well done, @mallardduck.

There is a [requirements doc](https://github.com/jfvoliveira/blade-boxicons#requirements) that I believe needs to be updated.

**Comment by Gummibeer** [comment]
@scrlkx only if this PR doesn't get merged. 🤔😔

**Comment by mallardduck** [comment]
Created a fork here to get a stable tagged release cut for the package: https://packagist.org/packages/mallardduck/blade-boxicons

If the original maintainer starts back up again that's cool. But for now we can keep projects wanting to use this moving.

**Pull request closed without merging**
